docs/guides/COMMIT_LEVELS.md

```diff
@@ -35,7 +35,7 @@ The development team has defined three (3) Github levels of **commit authorizati
   * Should ensure their issues are tested with latest HEAD versions of Angular Material
   * Should ensure their issues are tested with latest releases of Angular (1.3.x, 1.4.x, 1.5.x)
 * Core: 
-  * Includes: [Ryan Schmukler](https://github.com/rschmukler), [Robert Messerlee](https://github.com/robertmesserle), [Topher Fangio](https://github.com/topherfangio)
+  * Includes: [Ryan Schmukler](https://github.com/rschmukler), [Robert Messerle](https://github.com/robertmesserle), [Topher Fangio](https://github.com/topherfangio)
```

Comment:
Please remove Ryan and Robert... they are no longer active and should be removed from this note.